content/docs/sidebar.json

```diff
@@ -37,6 +37,7 @@
   {
     "slug": "start",
     "label": "Get Started",
+    "source": "start/index.md",
```

Comment:
Without this the [Edit on GitHub] button doesn't work for https://dvc.org/doc/start.